_test/check-exercises.escript

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env escript
+
+main( [] ) ->
+	Examples = filelib:wildcard( "*/example.erl" ),
+	Modules = [{X, compile(X)} || X <- Examples],
+	[compile_tests(X) || X <- Modules],
+	Results = [run_tests(X) || X <- Modules],
+	erlang:halt( erlang:length([X || X <- Results, X =/= ok]) );
+main( _ ) -> usage().
+
+compile( File ) ->
+	Compile = compile:file( File, [binary, return_errors] ),
+	{compile, File, {ok, Module, Binary}} = {compile, File, Compile},
+	Load = code:load_binary( Module, File, Binary ),
+	{load, Module, Load} = {load, Module, Load},
+	{Module, Binary}.
+
+
+compile_tests( {Example, {Example_module, _Binary}} ) ->
+	Filename = erlang:atom_to_list(Example_module) ++ "_tests.erl",
+	Filepath = filename:join( [filename:dirname(Example), Filename] ),
+	compile( Filepath ).
+
+
+run_tests( {_Example, {Module, _Binary}} ) ->
+	io:fwrite( "~p: ", [Module] ),
+	eunit:test( Module ).
```

Comment:
Consider changing this to verbose `eunit:test( Module , [verbose]).`
http://www.erlang.org/doc/man/eunit.html#test-1

Verbose
```bash
...
  All 10 tests passed.
word_count: ======================== EUnit ========================
module 'word_count'
  module 'word_count_tests'
    word_count_tests: count_one_word_test...ok
    word_count_tests: count_one_of_each_test...ok
    word_count_tests: count_multiple_occurrences_test...ok
    word_count_tests: ignore_punctuation_test...ok
    word_count_tests: include_numbers_test...ok
    word_count_tests: normalize_case_test...ok
    word_count_tests: prefix_punctuation_test...ok
    word_count_tests: symbols_are_separators_test...ok
    [done in 0.023 s]
  [done in 0.023 s]
=======================================================
  All 8 tests passed.
```
vs 
```bash
...
word_count:   All 8 tests passed.
```